### PR TITLE
OP-1290: Testing Coverage Improvements for ServicePrinting module

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-1290

### CODE REVIEW

File: rpt_base/TestReport.jasper
Comment (mwithi) [inline rpt_base/TestReport.jasper:1]: do you have also the corresponding `.jrxml` file?

Comment (dbmalkovsky) [reply rpt_base/TestReport.jasper:1]: No; the test path doesn't need it.

Comment (mwithi) [reply rpt_base/TestReport.jasper:1]: ok, but if one wants to inspect and/or modify it in the future has to start from scratch?

Comment (dbmalkovsky) [reply rpt_base/TestReport.jasper:1]: Added the file.

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**